pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "azure-operator"
 	source      string = "https://github.com/giantswarm/azure-operator"
-	version            = "4.2.1-dev"
+	version            = "5.0.0-dev"
```

Comment:
I think we can now bump the major version since this PR will start migrating the `AzureConfig` into CAPI & CAPZ types so it would then activate the CAPI & CAPZ controllers as well and therefore start working towards :mage: